experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -364,10 +364,30 @@ function _demazure_operator(r::RootSpaceElem, w::WeightLatticeElem)
   end
 end
 
-function demazure_operator(r::RootSpaceElem, w::WeightLatticeElem)
-  return demazure_operator(r, Dict(w => 1))
-end
+@doc raw"""
+    demazure_operator(r::RootSpaceElem, w::WeightLatticeElem) -> Dict{WeightLatticeElem,<:IntegerUnion}
+    demazure_operator(r::RootSpaceElem, groupringelem::Dict{WeightLatticeElem,<:IntegerUnion}) -> Dict{WeightLatticeElem,<:IntegerUnion}
+
+Computes the action of the Demazure operator associated to the positive root `r` on the given element of the groupring $\mathbb{Z}[P]$.
```

Comment:
```suggestion
Computes the action of the Demazure operator associated to the positive root `r` on the given element of the group ring $\mathbb{Z}[P]$.
```

Reply:
(it would be far preferable if the input was an actual group ring element, but this may not currently be an option... though my understanding is that recently some code for this (group rings over larger groups via dices) has been added to Hecke by @thofma ?

Reply:
I did some tinkering with group algebras over the weight lattice (as an additive group) to represent characters a few weeks ago. This is indeed the preferred goal, but I wasn't satisfied with the usability of these objects yet (this is the main reason why these things are still in experimental).
This would be something I or @janikapeters could look into at some point in the future, and upstream all necessary improvements to Hecke. And if we then are satisfied with the result, we can convert both the input here and the outputs of all character-like functions.